src/external-server/external-server.ts

```diff
@@ -149,6 +149,7 @@ export abstract class ExternalServer {
     }
 
     private async isServerAlive() {
+        if (this.port === null) return false;
```

Comment:
Why is this necessary?

Reply:
starknet-docker-proxy is initialized with null port, it avoid one request with null port